script/tool/lib/src/build_examples_command.dart

```diff
@@ -16,7 +17,10 @@ import 'common/repository_package.dart';
 /// Key for APK.
 const String _platformFlagApk = 'apk';
 
+const String _pluginToolsConfigFileName = '.pluginToolsConfig.yaml';
```

Comment:
I don't think we can know what format to pick to avoid a format break, but ok. (For example, what if we discover that actually what we want is for this to be a list of buildFlags, so that you can try running with several configurations? So the root should be a list rather than a map? Or, maybe the root should be platforms, with buildFlags as the second level. Or...)

Reply:
But when your root is a map you can *always* change things without it being a hard breaking change, because you can just use a different root key for the new format. Then eventually you can remove support for the old key.

(In this case the cost of the breaking change would be pretty low since we only guarantee that the tool works for two repos that we control; maybe I'm over-engineering based on experiences with things where it's much harder when you can't do the incremental change via a map.)

Reply:
i think things are a lot less clear-cut in practice than this (e.g. changing from a list to a map is non-breaking too since you can just keep on supporting the old list mode, and sometimes a map's keyspace is complete, e.g. the semantics are that you do something for each key, in which case the old key would conflict with the new keyspace, etc), but it's not a big deal either way.